index.js

```diff
@@ -15,6 +15,8 @@ const platform = process.platform
 const { tidy, noop, isSudo } = require('./lib/util')
 
 module.exports = zeroEks
+// Expose for custom visualizations
+module.exports.ticksToTree = ticksToTree
```

Comment:
can you move this to a separate file, and maybe add a unit test (maybe using tap snapshots?)